### ISSUE TITLE
Help - please clarify needed roles for bulkloading part attributes

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#7130]
The part attribute loader asserts the following:

> This tool allows Arctos operators with [role(s)](https://arctos.database.museum/Admin/user_roles.cfm) (manage_records) to create and review Part Attributess.

but an operator (rpianka) with appropriate collection access and roles is unable to access the tool.

![image](https://github.com/ArctosDB/arctos/assets/5725767/883046c1-160c-4d8c-87f1-7505e25f7b9f)

Can anyone clarify what role is missing for this tool

**Comment by mkoo** [comment]
seems to be the correct ones with manage_records and manage_specimens 
but maybe something about parts -- try adding update_container?

**Comment by dustymc** [comment]
https://arctos.database.museum/directory.cfm?dv=table&access=all&directaccess=all or the directory editor will tell you everything you need to know. I updated access to manage_records.
